src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,70 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+
+import cn from 'classnames';
+
+import { Tab } from '../../types';
+
+type Props = {
+  onTabSelected: (tabNr: Tab) => void,
+  tabs: Tab[] | [],
+  selectedTabId: string,
+};
+
+export const Tabs: FC<Props> = ({
+  onTabSelected,
+  tabs,
+  selectedTabId,
+}) => {
+  const tabContent = tabs.find(tab => (
+    tab.id === selectedTabId
+  ))?.content;
+
+  const isIdCorrect:boolean = tabs.some((tab:Tab) => (
+    tab.id === selectedTabId
+  ));
```

Comment:
no need to indicate type if it is too obvious (in this case – some returns boolean)
```suggestion
  const isIdCorrect = tabs.some((tab:Tab) => (
    tab.id === selectedTabId
  ));
```
also, if you indicate types, put a space in between
```
const someVariable: boolean = ...

// NOT the following way

const someVariable:boolean = ...
```